draft-ietf-rtcweb-jsep.xml

```diff
@@ -225,7 +225,7 @@
     <section title="Semantics and Syntax" anchor="sec.semantics-and-syntax">
       <section title="Signaling Model" anchor="sec.signaling-model">
         <t>JSEP does not specify a particular signaling model or state
-        machine, other than the generic need to exchange SDP media
+        machine, other than the generic need to exchange SDP session
```

Comment:
That would work fine as far as I'm concerned (I was after changing "media description" to "session description" - does not really matter if there is "SDP" before or not).